js/DataSourceSorter.js

```diff
@@ -13,10 +13,10 @@ var DataSourceSorter = DataSourceIndexed.extend('DataSourceSorter', {
      * @param {number} columnIndex
      * @param {number} [direction=1]
      */
-    sortOn: function(columnIndex, direction) {
+    sortOn: function(columnIndex, direction, type) {
         var dataSource = this.dataSource,
             columnName = dataSource.getFields()[columnIndex],
```

Comment:
We should not be using `getFields` anymore unless we really do need an array of field names. Use `dataSource.schema[columnIndex].name` instead.